core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -124,6 +124,18 @@ private static String getUserFromAuthorization(String authorization) {
     return stripped.substring(0, colonIndex);
   }
 
+  private FileSystem getFileSystem(String authorization) {
+    final String user = getUserFromAuthorization(authorization);
+
+    if (user == null) {
```

Comment:
`getUserFromAuthorization()` sometimes returns `null` or `""` (empty string). Should `""` be handled here as well?

Reply:
Let me change them all to return null.